### ISSUE TITLE
Make the status menu panels accessible with the Orca screen reader in Mint MATE.

### ISSUE THREAD

**Issue by slannon97** [linuxmint/xapp#188]
### Distribution

Mint MATE 22.1

### Package version

2.8.8+xia

### Graphics hardware in use

Intel Haswell-ULT Integrated Graphics driver: i915

### Frequency

Always

### Bug description

When I press Control Alt Tab to switch to the bottom panel in Mint 22 MATE, the Orca screen reader doesn't read all the tray icons to me when I tab or arrow around with the arrow keys. It will read the clock, but many other items such as the icon for Wi-Fi or system updates do not read at all. Instead, Orca says panel or panel notification area panel.

### Steps to reproduce

Enable Orca with Alt-Super-s.
Press Control-Alt-d to move focus to the desktop.
Press control-alt-tab to move to the bottom panel.
Explore the screen by pressing tab, shift-tab, and the arrow keys and listen to what Orca tells you.

### Expected behavior

Orca should clearly read what each control does, such as Wi-Fi, update manager status, etc. It should also be possible to use the space bar, enter, or the context menu with Shift-F10 or the Applications key to interact with these controls using only a keyboard.

### Additional information

This issue is still present in Mint 22.1. I hope it can be resolved soon, either by updating this particular component that handles tray icons, MATE, or both. It should be ideally be fixed by Mint 23 at the very latest. I want MATE in Linux Mint to be as accessible as possible. HEre's the original issue I opened which lead me here.
https://github.com/linuxmint/linuxmint/issues/693
